content/en/webassembly.md

```diff
@@ -0,0 +1,24 @@
+---
+title: WebAssembly
+status: Feedback Appreciated
+category: Concept
+tags: ["Application", "", ""]
+---
+
+## What it is
+
+WebAssembly (often abbreviated as Wasm) is a binary instruction format designed as a portable target for compiling high-level languages like C, C++, Rust, and others, enabling deployment on the web for client-side and server-side applications.
+It is a low-level bytecode format that can be executed in a virtual machine, typically integrated into web browsers.
+
+## Problem it addresses
+Traditional web applications are primarily built using JavaScript, which is an interpreted language.
+While JavaScript has improved over the years, it still faces performance limitations when executing computationally intensive tasks.
+With JavaScript, developers often need to rewrite their code to make it work across different platforms and environments. 
```

Comment:
I wonder about the rewriting bit here -- thinking more on it, one of the things that WASM helps with is not having to rewrite legacy code either -- if you have some old C code that is doing the job, it can potentially be used without rewriting on the web now too.

Reply:
I question the accuracy in comparing WASM to javascript at all. I would also remove this.